app/services/action_handlers/push.rb

```diff
@@ -0,0 +1,17 @@
+module ActionHandlers
+  class Push < ActionHandler
+    private
+
+    def handleable?
```

Comment:
What is this for?

Reply:
The superclass, `ActionHandler` has the `call` method defined like this:
```
def call
    return unless handleable?

    handle_action
end
```
The rest of the subclasses use it to check if the action is handleable, but for push events there's only one possible event, which is already being handled. So for them, this method should always return `true`